acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/web3j/generated/EventEmitter.java

```diff
@@ -146,18 +146,15 @@ protected EventEmitter(
     return web3j
         .ethLogFlowable(filter)
         .map(
-            new io.reactivex.functions.Function<Log, StoredEventResponse>() {
-              @Override
-              public StoredEventResponse apply(Log log) {
-                Contract.EventValuesWithLog eventValues =
-                    extractEventParametersWithLog(STORED_EVENT, log);
-                StoredEventResponse typedResponse = new StoredEventResponse();
-                typedResponse.log = log;
-                typedResponse._to = (String) eventValues.getNonIndexedValues().get(0).getValue();
-                typedResponse._amount =
-                    (BigInteger) eventValues.getNonIndexedValues().get(1).getValue();
-                return typedResponse;
-              }
+            log -> {
+              Contract.EventValuesWithLog eventValues =
+                  extractEventParametersWithLog(STORED_EVENT, log);
+              StoredEventResponse typedResponse = new StoredEventResponse();
+              typedResponse.log = log;
+              typedResponse._to = (String) eventValues.getNonIndexedValues().get(0).getValue();
+              typedResponse._amount =
+                  (BigInteger) eventValues.getNonIndexedValues().get(1).getValue();
+              return typedResponse;
```

Comment:
this was a suggested change by intellij, not sure if we wanted the explicitness of the complete anonymous class decl.

Reply:
From intellij:
Reports all anonymous classes which can be replaced with lambda expressions.
Note that if an anonymous class is converted into a stateless lambda, the same lambda object can be reused by Java runtime during subsequent invocations. On the other hand, when an anonymous class is used, separate objects are created every time. Thus, applying the quick-fix can cause the semantics change in rare cases, e.g. when anonymous class instances are used as HashMap keys.
Lambda syntax is not supported under Java 1.7 or earlier JVMs.

Reply:
+1 to using lambdas when possible.  So much easier to read.